tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php

```diff
@@ -1281,6 +1281,15 @@ class Sample
 }',
                 ['elements' => ['const' => 'none']],
             ],
+            'multiple trait import 5954' => [
+                '<?php
+class Foo
+{
+    use Bar, Baz;
+}',
+                null,
```

Comment:
can we also have a case when fixer does some change too?

Reply:
Done.